<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(131, 180, 226);
border-style: outset;
border-width: 0px;
border-radius: 10px;
font: bold 14px; 
</string>
    </property>
    <property name="text">
     <string>Add employee</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>      ID</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>      First Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>      Last Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>      National ID Number</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>      Birthday</string>
    </property>
   </widget>
   <widget class="QPushButton" name="OK">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>390</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(131, 180, 226);
border-style: outset;
border-width: 0px;
border-radius: 10px;
font: bold 14px; 
</string>
    </property>
    <property name="text">
     <string>OK </string>
    </property>
   </widget>
   <widget class="QPushButton" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>270</y>
      <width>291</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
